test/lib/mayaUsd/render/mayaToHydra/cpp/testUsdStageLayerMuting.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import maya.cmds as cmds
+
+import fixturesUtils
+import mtohUtils
+import unittest
+
+from testUtils import PluginLoaded
+
+class TestUsdStageLayerMuting(mtohUtils.MayaHydraBaseTestCase):
+    # MayaHydraBaseTestCase.setUpClass requirement.
+    _file = __file__
+
+    CUBE_PRIM_PATH = "/USDCube"
+    SUB_LAYER_CUBE_SIZE = 8.0
+
+    def setSubLayerMuted(self, isMuted: bool):
+        import maya.cmds as cmds
+        cmds.mayaUsdLayerEditor(self.subLayer.identifier, edit=True, muteLayer=(isMuted, self.usdProxyShapeUfePathString))
+
+    def setupUsdStage(self):
+        import maya.cmds as cmds
```

Comment:
Nit-pick: importing maya.cmds twice, may as well factor it out.

Reply:
Thanks, that import was even already factored out at the top so those were useless.